packages/react-devtools-shared/src/devtools/views/Profiler/SidebarSelectedFiberInfo.js

```diff
@@ -167,7 +167,7 @@ function WhatChanged({
   if (changeDescription.didHooksChange) {
     changes.push(
       <div key="hooks" className={styles.WhatChangedItem}>
-        • Hooks changed
+        • Internal state changed
```

Comment:
Slight preference for "hooks" since people know what that means at least. "Internal state" sounds like it might refer to React internals or something else.

Reply:
What about just "State changed"? What I found confusing was that changes due to `useContext` won't show up as "Hooks changed".

Reply:
Yeah, that's understandable. Most hooks aren't stateful in a way that triggers re-renders.

I'm not sure if "state changed" is better than "hooks changed" to be honest.